pages/glossary/focused.md

```diff
@@ -3,8 +3,8 @@ title: Focused
 key: focused
 unambiguous: true
 objective: true
+input_aspects:
+  - CSS styling
 ---
 
-An element is focused when that element matches the [`:focus` pseudo-class](https://drafts.csswg.org/selectors-4/#focus-pseudo).
-
-For HTML documents, see how the [`:focus` pseudo-class selector matches HTML elements](https://html.spec.whatwg.org/multipage/semantics-other.html#selector-focus).
```

Comment:
Why removing this precision?

Reply:
It got lost while resolving a conflict. Thanks for spotting that.